Fix CoC link and new container tab tour

hoelzro opened the pull request:
Hello there!

I noticed while reading the recent changes for testpilot-containers that the way to open a new tab in a container changed from hover to long press and that the tour didn't reflect that, so I fixed the tour.  I also noticed while going over the contributing document that the link to the CoC was broken, so I fixed that as well.

hoelzro committed:
Update tour for containers plugin

Hovering over the new tab button is no longer the way to activate
the container selection UI - it's long click as of version 2.4.0

hoelzro committed:
Fix link to code of conduct

chuckharmston commented on content-src/experiments/containers.yaml:
```diff
@@ -58,7 +58,7 @@ tour_steps:
   -
     image: /static/images/experiments/containers/tour/tour2.jpg
-    copy: Hover over the New Tab button to select a container.
+    copy: Long press on the New Tab button to select a container.
```
The build is failing because you changed the meaning of this string without changing the L10n ID, so translations won't be updated to reflect the new meaning. More info about [how we do localization](https://github.com/mozilla/testpilot/blob/master/docs/content/reference.md#localization).

You'll want to add something like:

```yaml
copy_l10nsuffix: 'longpress'
```

Then run `npm run l10n:extract` so that [`experiments.ftl`](https://github.com/mozilla/testpilot/blob/master/locales/en-US/experiments.ftl) has the new string and ID.

chuckharmston replied:
You can see what that might look like in #2646

hoelzro replied:
Thanks for taking a look so quickly!  I'll make that tweak right away!

hoelzro replied:
@chuckharmston The string change has been made - let me know if I got it right!

chuckharmston replied:
This looks exactly right! r+ from me, we'll wait to merge this until it's reviewed by our head localizer, who reviews all string changes.

chuckharmston reviewed (changes requested):
Thanks so much for the contribution! One small tweak and this will be good to merge.

hoelzro committed:
Tweak l10n for long press string

Based on comments in PR:

https://github.com/mozilla/testpilot/pull/2645#discussion_r128904136
